### ISSUE TITLE
Biochemistry Structures

### ISSUE THREAD

**Issue by pfrybar** [ModelSEED/ModelSEED#115]
It looks like you removed the biochemistry structures from the most recent Biochemistry object.. Do you have the corresponding Provenance object for these structures?

**Comment by cshenry** [comment]
"BiochemistryStructures" is the object that will contain the biochemistry structures now.

**Comment by pfrybar** [comment]
Right, do you have a BiochemistryStructures object that contains the structure data for the default biochemistry?

**Comment by cshenry** [comment]
Paul,

Ah, I see. Download the maindb folder with the source flat files from here:
http://bioseed.mcs.anl.gov/~chenry/ModelSEED/maindb.tgz
Then run the biochemistry and mapping import functions on this folder (follow the instructions on loading test database to see how).
This will generate a biochemistry structure for you. I of course, need to post mine to the download site, but this will take longer for me to figure out.

Chris
